libevm/ExtVMFace.h

```diff
@@ -239,8 +239,15 @@ class ExtVMFace
     /// Does the account exist?
     virtual bool exists(Address) { return false; }
 
-    /// Suicide the associated contract and give proceeds to the given address.
-    virtual void suicide(Address) { sub.suicides.insert(myAddress); }
+    /// Selfdestruct the associated contract and give proceeds to the given address.
+    ///
+    /// @param beneficiary  The address of the account which will receive ETH
+    ///                     from the selfdestructed account.
+    virtual void selfdestruct(Address beneficiary)
```

Comment:
Should we leave this arg unnamed? It would enable us to skip adding the (void)beneficiary...is there a reason to name it?